### PR TITLE
tic-tac-toe

### PR DESCRIPTION
**Pull request by oluwoleilesanmi**
Kindly help review.

### CODE REVIEW

File: assests/js/main.js [hunk 1]
```diff
@@ -0,0 +1,203 @@
...
+    playerA.setSymbol(SymbolGen.getRandSymbol());
+    playerA.getSymbol() === Consts.playerIdx ? playerB.setSymbol(Consts.playerIdo) : playerB.setSymbol(Consts.playerIdx);
+  };
+
+  return { attachboardListener, attachSymbol, reset, initialize, removeboardListener };
+})();
+
+
+let Board = (() => {
+  let tiles = ["", "", "", "", "", "", "", "", ""];
```
Comment (heyapricot) [inline assests/js/main.js]: I don't think you need to store empty string.
Also, you could use:
[Array.fill](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Array/fill)
or
`Array.from(Array(6))`

File: assests/js/main.js [hunk 2]
```diff
@@ -0,0 +1,203 @@
...
+    
+  let isTie = () => {
+    return !isWinner() && !allTilesFilled() ? true : false;
+  };
+
+  let allTilesFilled = () => {
+    let bol = false;
+    tiles.forEach(item => {
+      if (item === "") bol = true;
+    });
```
Comment (heyapricot) [inline assests/js/main.js]: Try with [Array.some()](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Array/some) instead

File: assests/js/main.js [hunk 3]
```diff
@@ -0,0 +1,203 @@
...
+    return bol;
+  };
+
+  let resetBoard = () => {
+    for (let index = 0; index < Consts.arrLen; index++) {
+      tiles[index] = "";
+    }
+  };
```
Comment (heyapricot) [inline assests/js/main.js]: What if you just make `tiles = Array.from(Array(9))`

File: assests/js/main.js (hunk 2 above)
Comment (oluwoleilesanmi) [reply assests/js/main.js]: Closed

File: assests/js/main.js (hunk 3 above)
Comment (oluwoleilesanmi) [reply assests/js/main.js]: Closed

**Pull request merged**